avm/src/vm/run.rs

```diff
@@ -106,6 +113,12 @@ pub async fn run_agz_b64(agz_b64: &str, priv_key_b64: Option<&str>, cert_b64: Op
   let mut gz = GzDecoder::new(bytes.as_slice());
   gz.read_i64_into::<LittleEndian>(&mut bytecode).unwrap();
   if priv_key_b64.is_some() && cert_b64.is_some() {
+    // Spin up a control port if we can start a secure connection
+    make_server!(HttpType::HTTPS(HttpsConfig {
+      port: 4142, // 4 = A, 1 = L, 2 = N (sideways) => ALAN
+      priv_key_b64: priv_key_b64.unwrap().to_string(),
+      cert_b64: cert_b64.unwrap().to_string(),
+    }), control_port);
```

Comment:
this is very daemon specific now. let's call this function `run_daemon` or this specific logic should probably be in `daemon.rs` or we should move the entire function to `daemon.rs`

Reply:
I do agree that this stuff feels like it should be moved into the daemon. I'm not yet sure how I want to do it, though? Can that be a refactor in a follow-up PR, since I want to fast-follow multiple PRs this afternoon on this? :)

Reply:
I am okay with it happening in a follow up PR